java/src/main/java/com/genexus/GxImageUtil.java

```diff
@@ -165,6 +166,57 @@ public static String flipVertically(String imageFile) {
 		return "";
 	}
 
+	public static String roundBorders(String imageFile, int topLeftRadius, int topRightRadius, int bottomLeftRadius, int bottomRightRadius) {
+		if (!isValidInput(imageFile)) return "";
+		try {
+			// Rounded images are basically images with transparent rounded borders and jpg and jpeg formats do not
+			// support transparency, so we have to create a new identical image but in png format
+			BufferedImage originalImage;
+			if (imageFile.indexOf(".jpg") > 0 || imageFile.indexOf(".jpeg") > 0){
+				originalImage = createBufferedImageFromURI(imageFile);
```

Comment:
You can create de originalImage before de if statement and not wirte de else part becuase it do the same.